scripts/cibuild.rb

```diff
@@ -34,9 +34,16 @@ def verify_file (f)
 
     dotNetInQuote = contents.index("- .NET")
 
+    multiLineEntry = contents =~ /(^\s?.*\n+$)/
```

Comment:
This seems to be looking at the whole YAML file. Should this just be the value of `desc`? I think we have to do this before parsing...